### ISSUE TITLE
[Oops.] CONSTRUCTSUBSET algorithm bug

### ISSUE THREAD

**Issue by Joshverge** [jeffgerickson/algorithms-ftw#182]
**Please verify that the error is present in the most recent revision before reporting.**
->Present in 1st edition and Chapters [Jun 15 2019]
**Chapter number or note title:** [Chp 2: Backtracking]
**Page number:** [79]
**Error description:** [uses wrong variable]
**Suggested fix (if any):** [change n to i]

**Comment by echuber2** [comment]
Hm, can you be more specific about what the problem is? This is what I see:

![construct-subset](https://user-images.githubusercontent.com/5503940/66699779-2ecb0600-ecaf-11e9-9fd3-876739f9ff30.png)

**Comment by echuber2** [comment]
Ah, now I see the *n=0* part.

**Comment by Joshverge** [comment]
Yeah I guess I should have mentioned the line number or something but since there is only one n present I didn't think about that.
